### DISCUSSION TITLE
Can we use pnpm?

### DISCUSSION THREAD

**Discussion by spences10** [jycouet/kitql#46] [General]
Hi, I really like using pnpm in place of npm and yarn now

what would it take to have kitql compatible with pnpm?

**Comment by jycouet** [comment]
I updated the `demo1` example to be able to use `pnpm`. 👍
You can give a try now

**Reply by spences10** [reply]
Looks like it uses [yarn] in `svelte.config.js`, is that intentional?

[yarn]: https://github.com/jycouet/kitql/blob/main/examples/demo1/svelte.config.js#L19
